uikit/common/src/main/java/com/sygic/ui/common/extensions/UIExtensions.kt

```diff
@@ -1,19 +1,19 @@
 package com.sygic.ui.common.extensions
 
-import android.graphics.drawable.Drawable
-import android.graphics.drawable.TransitionDrawable
-import android.widget.ImageView
+import android.util.TypedValue
+import android.view.View
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
+import androidx.annotation.StyleRes
 
-fun ImageView.setImageDrawableWithFade(targetDrawable: Drawable, duration: Int) {
-    val currentDrawable = this.drawable
-    if (currentDrawable == null) {
-        this.setImageDrawable(targetDrawable)
-        return
-    }
+fun View.applyStyle(@StyleRes resId: Int, force: Boolean = false) {
+    context.theme.applyStyle(resId, force)
+}
 
-    val layers = arrayOf(currentDrawable, targetDrawable)
-    val transition = TransitionDrawable(layers)
-    transition.isCrossFadeEnabled = true
-    this.setImageDrawable(transition)
-    transition.startTransition(duration)
+@ColorInt
+fun View.getColorFromAttr(@AttrRes resId: Int, typedValue: TypedValue = TypedValue(), resolveRefs: Boolean = true): Int {
```

Comment:
ono to funguje vseobecne na vsetky attributy, nie len farby. takze ten nazov je trochu zavadzajuci :) poslat ti tam moze niekto aj attr/dimen a hodnotu to resolvne, len to nebude farba ;)

Reply:
pravdu máš, přejmenuji ;)